caikit/runtime/service_factory.py

```diff
@@ -160,6 +162,7 @@ def get_service_package(cls, service_type: ServiceType) -> ServicePackage:
             registration_function=grpc_service.registration_function,
             stub_class=grpc_service.client_stub_class,
             messages=client_module,
+            caikit_rpcs=set(task_rpc_list),
```

Comment:
nit: `task_rpc..` here seems confusing if valid for both inference and training

Reply:
oops, I had made that change in my other in-flight PR, you good with merging this and I'll resolve the conflict on that one?

Reply:
that is acceptable